docs/sources/enterprise/reporting.md

```diff
@@ -42,6 +42,16 @@ Currently only Organization Admins can create reports.
 
 {{< docs-imagebox img="/img/docs/enterprise/reports_create_new.png" max-width="500px" class="docs-image--no-shadow" >}}
 
+## Send test mail
```

Comment:
Try rewriting this as a numbered steps. The task is Send a test mail.

1. In the (place), click **Send test mail**.
1. In the (field name), enter the email address or addresses that you want to test. (Do you need to separate them by a comma?
  If you want to use email addresses from the report, then select the **Use emails from report** check box.

Reply:
Thanks!